azext_iot/central/command_map.py

```diff
@@ -91,6 +91,13 @@ def load_central_commands(self, _):
         cmd_group.command("run", "run_command")
         cmd_group.command("history", "get_command_history")
 
+    with self.command_group(
+        "iot central device compute-device-key",
```

Comment:
nit - i thought we settled on `az iot central device key create` for central

Reply:
keeping it consistent across dps and central. worked with the pm.